concent_api/conductor/models.py

```diff
@@ -77,6 +77,9 @@ class OutputFormat(ChoiceEnum):
     # Relative path to the .blend file inside the source package.
     scene_file = CharField(max_length=MESSAGE_PATH_LENGTH)
 
+    # Relative path to the Python script to be executed by Blender.
```

Comment:
it is not a path - it is the content of the script itself

Reply:
Exactly. Please look at #520, it's all described in detail there. In particular the sample message linked there (https://github.com/golemfactory/concent/issues/235#issuecomment-378525017) shows the content of the `TaskToCompute.script_src` field which is what is passed to `blender_crop_script`.

Reply:
So what does it mean 
> Remove the script after rendering to make sure that the next task does not reuse it.

It means Subtask object field should be cleared ?

Reply:
Please look at `src_code` in `TaskToCompute` (that's the other script, not the one we put in `blender_crop_script`). This script creates a file called `WORK_DIR + "/blenderscript.py"` and puts `script_src` in it. Our code for running blender is based on it so I assume you'll be doing something similar. This remark was just to make sure that you do not forget to remove this temporary file when Blender exits.

Reply:
:+1: done, but I think it would be good to add this info in a more direct way to the issue.

Reply:
I could, but honestly I think that issues are very detailed already. This is a detail that becomes apparent once you try to implement it. One look at the content of `TaskToCompute.script_src` is enough to realize that it's not a path. At worst, it would be discovered once you try to run it after implementing it and realize that it tries to use the script as if it were a path.

Actually, how did it even work for you? It could not have worked as implemented originally.

Reply:
> One look at the content of TaskToCompute.script_src is enough to realize that it's not a path

I didn't thought it is a path, but that it might be passed directly as a string.

> Actually, how did it even work for you? It could not have worked as implemented originally.

I didn't tested it yet, can it be tested locally ?

Reply:
> I didn't thought it is a path, but that it might be passed directly as a string.

It is passed directly as a string. `blender_crop_script` is the whole script, stored directly in a string.

> I didn't tested it yet, can it be tested locally ?

Yes. Just like any other part of verification. For example the blender rendering task is not closed yet because @dybi is (was?) testing it with sample projects.

Reply:
> It is passed directly as a string. blender_crop_script is the whole script, stored directly in a string.

Ok, so first I though it is a path to one of the files from archives. I have checked in `TaskToCompute`. It looked like something which is not being set at the moment. Then during review I found out it is a script, so I though the script code might be passed directly to Blender command. As I understand now, it should be first written to a file, and that file name should be passed to Blender. This is the info I consider worth mentioning in description.

@dybi Are you able to test this one too or provide TL:DR how to do this locally ?